src/Compilers/CSharp/Test/Syntax/Parsing/TopLevelStatementsParsingTests.cs

```diff
@@ -1610,26 +1610,16 @@ public void NewKeyword()
         }
 
         [Fact]
-        public void TupleUnsupportedInUsingStatement()
+        public void TupleUnsupportedInUsingStatement_CSharp9()
```

Comment:
>public void TupleUnsupportedInUsingStatement_CSharp9() [](http://example.com/codeflow?start=8&length=54)

Should the results of this test be identical between C#9 and preview? If so, please consider combining into a single test (and renaming the test to remove "Unsupported"):
```csharp
[Theory]
[InlineData(LanguageVersion.CSharp9)]
[InlineData(LanguageVersion.Preview)]
public void TupleInUsingStatement(LanguageVersion langVersion)
{
    var test = "...";
    UsingTree(test, langVersion));
    ...
```